signify-ts-test/src/utils/handle-json-config.ts

```diff
@@ -20,20 +23,62 @@ export async function buildUserData(secretsJson: any): Promise<Array<User>> {
   return users;
 }
 
+export async function buildCredentials(
+  secretsJson: any,
+): Promise<Map<string, CredentialInfo>> {
+  let credentials: Map<string, CredentialInfo> = new Map<
+    string,
+    CredentialInfo
+  >();
+  for (const key in secretsJson.credentials) {
+    const cred = secretsJson.credentials[key];
+    let curCred: CredentialInfo = {
+      type: cred.type,
+      issuer: cred.issuer,
+      issuee: cred.issuee,
+      recipient: cred.recipient,
+      schema: cred.schema,
+      rules: cred.rules,
+      privacy: cred.privacy,
+      attributes: cred.attributes,
+      credSource: cred.credSource,
+    };
+    credentials.set(key, curCred);
+  }
+  return credentials;
+}
+
 export interface User {
   secrets: Map<string, string>;
   LE: string;
   clients: Map<string, Array<SignifyClient>>;
   aids: Map<string, Array<any>>;
   oobis: Map<string, Array<any>>;
+  contextRole: string;
+  alias: string;
+}
+
+export interface CredentialInfo {
+  type: string;
+  issuer: string;
+  issuee: string;
+  recipient?: string;
+  schema: string;
+  rules?: string;
+  privacy: boolean;
+  attributes: any;
+  credSource?: any;
 }
 
 interface SecretUser {
```

Comment:
eventually migrate away from the concept/naming of 'SecretUser' as the configuration is no longer just secrets.